### PR TITLE
Vidoe Player  in Music Tab

### PR DESCRIPTION
**Pull request by MonCaptain**
Works well with just a single video. Inconsistent when it comes to playlist videos.

### CODE REVIEW

File: react-app/pomodoro-timer-ui/src/Components/MusicTab/VideoPlayer.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import "./MusicTab.css";
+import { useState } from "react";
+import ReactPlayer from 'react-player/youtube'
+import {useMusicContext} from "../../contexts/MusicContext"
+
+export default function VideoPlayer({displayType}) {
+  const {MusicContextVariables} = useMusicContext()
+  const videoUrl = MusicContextVariables.urlForm
+  console.log(1)
```
Comment (Rebecca-CYLi) [inline react-app/pomodoro-timer-ui/src/Components/MusicTab/VideoPlayer.jsx]: found a console log

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
Found a console log, everything else looks good

**Review by Rebecca-CYLi — APPROVED** [review]
no console logs, LGTM

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
oops, I misread lines. still needs to fix console log

**Comment by MonCaptain** [comment]
> oops, I misread lines. still needs to fix console log

I pushed new changes into the branch after your approval, will look into them.

**Review by Rebecca-CYLi — APPROVED** [review]
revisions made, LGTM

**Comment by MonCaptain** [comment]
> revisions made, LGTM

Thank you thank you

**Pull request merged**